ENV_SETUP.md

```diff
@@ -0,0 +1,16 @@
+** the script will set up the env for compiling - no need to run twice ** 
+** to test new changes, simply enter container shell and execute scons - the files will update automatically due to mounitng feature ** 
+** this also means that, once built, the build dir will be on your local machine as well. Delete if you want to re-build from scratch **
+
+Local changes 
+- Clone repo
+	- clone with ssh link to enable ssh connection when pulling and pushing 
+- Pull in submodules
+	- git submodule --update --init recursive 
+- Install and start docker (OS-specific)
+- Execute compile script from dev_container dir
+	- ./compile_docker.sh
```

Comment:
hmmmm I'm just now realizing this won't work nicely on windows unless you're using WSL2. Probably fine to proceed but might be worth noting

Reply:
ya was worried about this too for windows users. seems like theres no way of running bash without WSL. should we add it as a requirement?

Reply:
you could also do it in a cygwin or mingw64 shell I think, in which case my previous comment might be wrong.